api/v1/models/status_response.go

```diff
@@ -104,6 +104,9 @@ type StatusResponse struct {
 
 	// List of stale information in the status
 	Stale map[string]strfmt.DateTime `json:"stale,omitempty"`
+
+	// Status of Authentication Spire connection
+	Spire *Status `json:"spire,omitempty"`
```

Comment:
> This file was generated by the swagger tool.

-> you need to edit the respective `openapi.yaml` and generate the code with `make generate-api generate-health-api generate-hubble-api generate-operator-api`